### PR TITLE
Migrate MySQL virtual schema

### LINKED ISSUE #1: Migrate MySQL virtual schema from virtual-schemas repo

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
Closes #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+## Features
+
+* Access a MySQL data source in read only mode from an Exasol database, using a Virtual Schema.
+
+## Table of Contents
+
+### Information for Users
+
+* [Virtual Schemas User Guide][user-guide]
+* [MySQL Dialect User Guide](doc/dialects/mysql_user_guide.md)
```
Comment (jakobbraun) [inline README.md:35]: `dialects` doesn't make sens any more

Comment (jakobbraun) [reply README.md:35]: I think we had a nice solution for postgresql

File: doc/changes/changes_1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#Virtual Schema for MySQL 1.0.0, released 2021-??-??
```
Comment (jakobbraun) [inline doc/changes/changes_1.0.0.md:1]:
```suggestion
# Virtual Schema for MySQL 1.0.0, released 2021-??-??
```

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+				<configuration>
+					<excludes>
+						<exclude>/doc/changes/*</exclude>
+					</excludes>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>com.exasol</groupId>
+				<artifactId>project-keeper-maven-plugin</artifactId>
+				<version>0.4.1</version>
```
Comment (jakobbraun) [inline pom.xml:299]: change to 0.4.2

**Review by jakobbraun — CHANGES_REQUESTED** [review]

**Comment by jakobbraun** [comment]
Nexted time let's try to do it like this: https://ariya.io/2014/07/extracting-parts-of-git-repository-and-keeping-the-history

**Review by jakobbraun — APPROVED** [review]

**Comment by chiaradiamarcelo** [comment]
> Nexted time let's try to do it like this: https://ariya.io/2014/07/extracting-parts-of-git-repository-and-keeping-the-history

sounds good! sure!

**Pull request merged**